.pre-commit-config.yaml

```diff
@@ -12,5 +12,27 @@
   - id: trailing-whitespace
     args: [--markdown-linebreak-ext=md, --markdown-linebreak-ext=feature]
   - id: check-json
+    exclude: .vscode/*
```

Comment:
vscode json files are not true json files, therefore I exclude them from the pre-commit checks